tutorial03/README.md

```diff
@@ -19,7 +19,7 @@ mkdir my-model-dir
 You have to let ED know where it will be able to find the models. You can do this by setting the ED_MODEL_PATH variable.
 
 ```bash
-export ED_MODEL_PATH=`pwd`/my-model-dir
+source ../setup_env.bash  # This should be done from the tutorial specific folder
```

Comment:
Is it a good idea to hide the internals in this tutorial? 
Its important that people can know how ED will find their models. granted, you can show it once and then use setup scripts like this in subsequent tutorials

Reply:
I think appending paths isn't that complex